tests/Functional/ContentPublicationRenderTest.php

```diff
@@ -26,6 +27,9 @@ class ContentPublicationRenderTest extends ContentRenderTestBase {
     'path',
     'oe_theme_helper',
     'oe_theme_content_publication',
+    'oe_theme_content_organisation',
+    'oe_content_organisation_reference',
```

Comment:
`oe_content_organisation_reference` can be removed because dependency exists in oe_theme_content_organisation_reference